test/unit/gateways/orbital_test.rb

```diff
@@ -174,6 +174,22 @@ def test_network_tokenization_credit_card_data
     end.respond_with(successful_purchase_response)
   end
 
+  def test_schema_for_soft_descriptors_with_network_tokenization_credit_card_data
+    options = @options.merge(
+      soft_descriptors: {
+        merchant_name: 'Merch',
+        product_description: 'Description',
+        merchant_email: 'email@example'
+      }
+    )
+    stub_comms do
+      @gateway.purchase(50, network_tokenization_credit_card(nil, eci: '5', transaction_id: 'BwABB4JRdgAAAAAAiFF2AAAAAAA='), options)
+    end.check_request do |endpoint, data, headers|
+      # Soft descriptor fields should come before dpan and cryptogram fields
+      assert_match %{<SDMerchantEmail>email@example</SDMerchantEmail> <DPANInd>Y</DPANInd> <DigitalTokenCryptogram}, data.squish
```

Comment:
The Travis CI does not like the use of `squish` here:
![image](https://user-images.githubusercontent.com/42787645/93250772-4b927800-f761-11ea-9fcd-ea0bdae8b602.png)

I'm not entirely sure if the CI will find this acceptable, but I got the test to pass using this format:
```suggestion
      data =~ /<SDMerchantEmail>email@example<\/SDMerchantEmail> <DPANInd>Y<\/DPANInd> <DigitalTokenCryptogram/
```

Reply:
updated! I ended up subbing all whitespace and then doing an assert_match (I was getting `nil` as the return value for `data =~ /<SDMerchantEmail>email@example<\/SDMerchantEmail> <DPANInd>Y<\/DPANInd> <DigitalTokenCryptogram/` rather than true, I think because there's some weird whitespace and line breaks in that data string)